src/crate/client/sqlalchemy/types.py

```diff
@@ -131,24 +132,31 @@ def __eq__(self, other):
         return dict.__eq__(self, other)
 
 
-class _Craty(sqltypes.UserDefinedType):
+class ObjectTypeImpl(sqltypes.UserDefinedType, sqltypes.JSON):
+
+    __visit_name__ = "OBJECT"
+
     cache_ok = False
```

Comment:
We still need to use `cache_ok = False`. It does not improve the situation wrt. GH-559.